src/main/java/xyz/vegaone/easytrackingv2/controller/ErrorController.java

```diff
@@ -0,0 +1,39 @@
+package xyz.vegaone.easytrackingv2.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+import xyz.vegaone.easytrackingv2.exception.UserNotFoundException;
+
+@Controller
+public class ErrorController {
```

Comment:
This class can be removed. The error handling is done in the user controller class.